device-types/Ubiquiti/UVC-G6-Turret.yaml

```diff
@@ -0,0 +1,87 @@
+---
+manufacturer: Ubiquiti
+model: Unifi G6 Turret
+slug: ubiquiti-unifi-uvc-g6-turret
+part_number: UVC-G6-Turret
+comments: |
```

Comment:
Thank you for your contribution,
Please reduce the amount of information in the comments property, this should only be a brief description with a link to the documentation.